### PR TITLE
[Temperature] Add support for CPUs with specific TjMax

### PR DESCRIPTION
**Pull request by x86fr**
Solve an issue where reading MSR_IA32_TEMPERATURE_TARGET makes the system crash (Celeron M 215 / Intel D201GLY board)

### CODE REVIEW

File: system/temperature.c [hunk 1]
```diff
@@ -33,58 +33,81 @@ float cpu_temp_offset = 0;
 // Public Functions
 //------------------------------------------------------------------------------
 
+int TjMax = 0;
```
Comment (debrouxl) [inline system/temperature.c:36]: +static ?

File: system/temperature.c [hunk 2]
```diff
@@ -33,58 +33,81 @@ float cpu_temp_offset = 0;
...
+
+uint32_t regl, regh;
```
Comment (debrouxl) [inline system/temperature.c:38]: Unless you need to export them later, why not keep them local variables ?

File: system/temperature.c (hunk 1 above)
Comment (x86fr) [reply system/temperature.c:36]: Yep!

File: system/temperature.c (hunk 2 above)
Comment (x86fr) [reply system/temperature.c:38]: 1 line saved 😅

Seriously, yes, will move them back.

**Pull request merged**